infinitest-lib/src/main/java/org/infinitest/parser/JavaAssistClass.java

```diff
@@ -409,4 +423,37 @@ public boolean locatedInClassFile() {
 	public File getClassFile() {
 		return classFile;
 	}
+
+	private static class SpockIsTestClassChecker {
```

Comment:
This class is quite big, so I created an inner class to (at least directly) do not enlarge it. Assuming, there is no way to avoid checking for testing frameworks in the lib module, probably it would be good to extract support for the particular frameworks to separate classes.